### ISSUE TITLE
livemarks not stored in folder as per settings

### ISSUE THREAD

**Issue by wallymann** [nt1m/livemarks#351]
**Add-on version**
Add-on version: 3.3

**Describe the bug**
added feeds and imported feeds are not saved in the correct folder as per the settings.

**To Reproduce**
1. create a bookmarks folder e.g., "Livemarks".
2. set default RSS folder in livemarks options as the "Livemarks" folder just created in step-1.
3. import a RSS OPML file (example attached) or subscribe to a new RSS feed.
4. you will see that the RSS feeds are *not* in the "Livemarks" folder as per the settings, instead they are all at the root bookmark folder.

**Expected behavior**
all RSS feeds should be stored in the chosen folder e.g., "Livemarks"

**Extra context**
![2021-04-12 09_57_40-Greenshot](https://user-images.githubusercontent.com/18448369/114408188-2b039280-9b77-11eb-8be9-420c5f527fab.jpg)
![2021-04-12 09_57_12-Window](https://user-images.githubusercontent.com/18448369/114408240-3656be00-9b77-11eb-9630-32721a009d47.jpg)
[feedbro-subscriptions-20210412-093136.opml.txt](https://github.com/nt1m/livemarks/files/6297313/feedbro-subscriptions-20210412-093136.opml.txt)

**Comment by evilpie** [comment]
Does this still happen?

**Comment by wallymann** [comment]
yes...the issue persists, i'm no longer able to effectively use livemarks.  any suggestions?

**Comment by Marakai** [comment]
Adding comment that this is also happening for me with the latest version. Old/pre-existing feed work fine, any new feed added show this behaviour.

**Comment by evilpie** [comment]
@Marakai do you have any other extension installed?

**Comment by Ric783** [comment]
Same here. Existing feeds work normally, but new feeds have no folder structure. Adding a new feed is not currently a useable function.

**Comment by wallymann** [comment]
@Ric783 -- interesting observation re: DBF addon.  i do not have this addon installed.

**Comment by Ric783** [comment]
Just to clarify, I am not sure whether I have quite the same problem as @wallymann. I am not changing the folder setting in Livemarks, and I am not clear whether @wallymann only has a problem when that default is changed.

The default Firefox folder for bookmarks has changed, though, which is why I use the DBF addon.

**Comment by evilpie** [comment]
Livemarks is not compatible with the extension "Default Bookmark Folder". The bookmark star in Firefox actually remembers the last selected folder as well.

**Comment by Ric783** [comment]
@evilpie  It used to, but now the folder has to be reselected every time - unless that's just for the Linux version. "Default Bookmark Folder" restores the old behaviour, as do other addons :)

Once the feed is saved, there is no problem with Livemarks.

**Comment by wallymann** [comment]
ok...i do have the DBF add-on...i will test disabling DBF when adding feeds to livemarks

**Comment by evilpie** [comment]
I don't think that is true, but anyway not important: https://bugzilla.mozilla.org/show_bug.cgi?id=1432604.

Is anyone experience this issue without "Default Bookmark Folder" being installed?

**Comment by wallymann** [comment]
ok...i tested the "disable DBF, import/add livemarks, re-enable DBF" and it's not really working.

the import *appears* to work, in that it creates the livemarks in my desired location.  see image 1 below.

however, all of the actual RSS updates are just dumped into the root bookmark folder.  see image 2 below.

![image](https://user-images.githubusercontent.com/18448369/136377493-8955cc1c-c85c-4be3-a8e2-aa36f85dc362.png)
![image](https://user-images.githubusercontent.com/18448369/136377513-33fd4f22-f0dd-4b6f-8e21-cb3f0c302b7d.png)

**Comment by evilpie** [comment]
This is expected. "Default Bookmark Folder" will move all bookmarks created by Livemarks. As I said both extensions are *incompatible*.
